### PR TITLE
perf(esbuild): don't keep names during minification

### PR DESCRIPTION
**Pull request by SevInf**
Saves 18KB. For error classes, names actually matter, so we do preserve
them manually. Has small impact on performance:

image

### CODE REVIEW

File: packages/client/src/runtime/core/errors/PrismaClientRustError.ts [hunk 1]
```diff
@@ -18,10 +20,12 @@ export class PrismaClientRustError extends Error {
   }
 
   get [Symbol.toStringTag]() {
-    return 'PrismaClientRustPanicError'
+    return 'PrismaClientRustError'
```
Comment (Jolg42) [inline packages/client/src/runtime/core/errors/PrismaClientRustError.ts:23]: Found a bug? 😄

File: packages/client/src/runtime/query.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import indent from 'indent-string'
 import { bold, dim, green, red, white } from 'kleur/colors'
 import stripAnsi from 'strip-ansi'
 
+import { setClassName } from '../../../internals/dist/utils/setClassName'
```
Comment (Jolg42) [inline packages/client/src/runtime/query.ts:6]:
```suggestion
import { setClassName } from '@prisma/internals/dist/utils/setClassName'
```
Like this does not work?

Comment (SevInf) [reply packages/client/src/runtime/query.ts:6]: Oops, VS Code auto-insertion

Comment (Jolg42) [reply packages/client/src/runtime/query.ts]: actually `import { setClassName } from '@prisma/internals'` would be the best?

File: packages/client/src/runtime/core/errors/PrismaClientRustError.ts (hunk 1 above)
Comment (janpio) [reply packages/client/src/runtime/core/errors/PrismaClientRustError.ts:23]: This might actually explain a few confusing issues we had in the past...

Comment (SevInf) [reply packages/client/src/runtime/core/errors/PrismaClientRustError.ts:23]: Yep, copy-paste from a sibling file, most likely.

File: packages/client/src/runtime/query.ts (hunk 2 above)
Comment (SevInf) [reply packages/client/src/runtime/query.ts]: fixed

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**